packages/apps/electron-app/src/cli/launcher.sh

```
@@ -4,26 +4,27 @@ set -eu
 source_path=$0
 
 while [ -L "$source_path" ]; do
-  source_dir=$(CDPATH= cd -P -- "$(dirname -- "$source_path")" >/dev/null 2>&1 && pwd)
+  source_dir=$(CDPATH= cd -P -- "$(dirname -- "$source_path")" > /dev/null 2>&1 && pwd)
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)